src/games/strategy/triplea/ui/RouteOptimizer.java

```diff
@@ -101,21 +160,20 @@ public static Point getPoint(Point2D point) {
       }
       int counter = 0;
       for (Point point : points) {
-        Point normalizedPoint = normalizePoint(point);
         if (isInfiniteX) {
-          alternativePoints.get(0)[counter] = new Point(normalizedPoint.x - mapWidth, normalizedPoint.y);
-          alternativePoints.get(1)[counter] = new Point(normalizedPoint.x + mapWidth, normalizedPoint.y);
+          alternativePoints.get(0)[counter] = new Point(point.x - mapWidth, point.y);
+          alternativePoints.get(1)[counter] = new Point(point.x + mapWidth, point.y);
         }
         if (isInfiniteY) {
           int index = altArrayCount == maxAdditionalScreens ? 2 : 0;
-          alternativePoints.get(index)[counter] = new Point(normalizedPoint.x, normalizedPoint.y - mapHeight);
-          alternativePoints.get(index + 1)[counter] = new Point(normalizedPoint.x, normalizedPoint.y + mapHeight);
+          alternativePoints.get(index)[counter] = new Point(point.x, point.y - mapHeight);
+          alternativePoints.get(index + 1)[counter] = new Point(point.x, point.y + mapHeight);
         }
         if (isInfiniteX && isInfiniteY) {
-          alternativePoints.get(4)[counter] = new Point(normalizedPoint.x - mapWidth, normalizedPoint.y - mapHeight);
-          alternativePoints.get(5)[counter] = new Point(normalizedPoint.x - mapWidth, normalizedPoint.y + mapHeight);
-          alternativePoints.get(6)[counter] = new Point(normalizedPoint.x + mapWidth, normalizedPoint.y - mapHeight);
-          alternativePoints.get(7)[counter] = new Point(normalizedPoint.x + mapWidth, normalizedPoint.y + mapHeight);
+          alternativePoints.get(4)[counter] = new Point(point.x - mapWidth, point.y - mapHeight);
+          alternativePoints.get(5)[counter] = new Point(point.x - mapWidth, point.y + mapHeight);
+          alternativePoints.get(6)[counter] = new Point(point.x + mapWidth, point.y - mapHeight);
```

Comment:
What do the indexes represent, how do we know we'll have 6, and what do they represent (please update the code to answer this)